src/main/java/build/buildfarm/common/config/Worker.java

```diff
@@ -40,6 +40,7 @@ public class Worker {
   private ExecutionPolicy[] executionPolicies = {};
   private SandboxSettings sandboxSettings = new SandboxSettings();
   private boolean createSymlinkOutputs = false;
+  private int zstdBufferPoolSize = 2048; /* * 128k == 256MiB */
```

Comment:
```suggestion
  private int zstdBufferPoolSize = 2048; /* A value of 128k will use up to 256MiB of RAM */
```
or otherwise clarify this comment?